src/utils/dom-utils/inviteToOpenSauced.ts

```diff
@@ -0,0 +1,36 @@
+import { InviteToOpenSaucedButton } from "../../components/InviteToOpenSauced/InviteToOpenSaucedButton";
+import { InviteToOpenSaucedModal } from "../../components/InviteToOpenSauced/InviteToOpenSaucedModal";
+import { getTwitterUsername, getLinkedInUsername } from "../matchers";
+
+const injectViewOnOS = (username: string) => {
```

Comment:
`injectOpenSaucedInviteButton`?
and similarly `injectViewOnOpenSaucedButton` below makes more sense.

Reply:
I agree with this suggestion. It provides clarity.

Reply:
Updated the ViewOnOpenSaucedButton here. https://github.com/open-sauced/browser-extensions/pull/20/commits/9e156f18de49a9092e921248bf4ee60ac4162d2d.